components/script/dom/trustedtypepolicyfactory.rs

```diff
@@ -73,13 +73,11 @@ impl TrustedTypePolicyFactory {
     ) -> Fallible<DomRoot<TrustedTypePolicy>> {
         // Step 1: Let allowedByCSP be the result of executing Should Trusted Type policy creation be blocked by
         // Content Security Policy? algorithm with global, policyName and factory’s created policy names value.
+        let policy_names = self.policy_names.borrow();
```

Comment:
Whoops, this borrow is still active when we do `self.policy_names.borrow_mut().push(policy_name);` later.

Reply:
Ugh. Yet when I did clone, the compiler said that the value doesn't live long enough as it is a `&[&str]`. Do you have any guidance how I should deal with these lifetime compilation issues?